docs/tutorials/circuits_advanced/06_building_pulse_schedules.ipynb

```diff
@@ -310,12 +310,12 @@
    "source": [
     "#### Pulse library functions\n",
     "\n",
-    "Our own pulse library has sampling methods to build a `Waveform` from common functions."
+    "It is possible to convert `SymbolicPulse` objects into `Waveform`s via the `get_waveform()` method. Using our own pulse library you can build `Waveform` from common functions."
```

Comment:
fyi @abbycross

Reply:
@nkanazawa1989 can you tell me if any (other) information in the Building Pulse Schedules tutorial is important to migrate to the Qiskit docs on IBM Quantum? I believe it will get lost otherwise with the big migration.

Reply:
I think new docs is largely based off of the existing Building Pulse Schedules tutorial so the rest of contents should be fine.